apps/wallet/src/shared/plausible.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import Plausible from 'plausible-tracker';
+
+// NOTE: The url of Sui wallet Chrome extension:
+// https://chrome.google.com/webstore/detail/sui-wallet/opcgpfmipidbgpenhmajoajpbobppdil
+const WALLET_URL = 'chrome-extension://opcgpfmipidbgpenhmajoajpbobppdil';
```

Comment:
Why not use [runtime.getUrl('')](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getURL)?

Reply:
Yeah good question. The dev runtime URL is different, but I'm not sure how much that matters. We could separate out the dev Plausible as well (probably a good idea).

Reply:
I mentioned also in case we publish a ff package as well to automatically handle the url. (I would expect an env tag for plausible in order to specify the env - I think sentry has something like that).

Reply:
Updated to use this